test/test_account_verification.js

```diff
@@ -0,0 +1,40 @@
+const _deploy_contracts = require("../migrations/2_deploy_contracts");
+const truffleAssert = require("truffle-assertions"); // npm truffle-assertions
+const BigNumber = require("bignumber.js"); // npm install bignumber.js
+var assert = require("assert");
+
+var Platform = artifacts.require("../contracts/Platform.sol");
+var Account = artifacts.require("../contracts/Account.sol");
+let accountOwner;
+let platformOwner;
+contract("AccountVerification", function (accounts) {
+    before(async () => {
+        accountOwner = accounts[0];
+        platformOwner = accounts[1];
+        accountInstance = await Account.deployed({from:accountOwner});
+        platformInstance = await Platform.deployed({from:platformOwner});
```

Comment:
By default all contract should be deployed under accounts[0].  Can we standardise account management, accounts[0] owner of contracts, accounts[1] seller/organizer, accounts[2,3,...] buyers